dlt/_workspace/helpers/dashboard/utils.py

```diff
@@ -74,6 +77,56 @@ def _exception_to_string(exception: Exception) -> str:
     return str(exception)
 
 
+def sync_from_runtime() -> None:
+    """Sync the pipeline states and traces from the runtime backup, recursively."""
+    from dlt.pipeline.runtime_artifacts import _get_runtime_artifacts_fs
+    import fsspec
+
+    def sync_dir(fs: fsspec.filesystem, src_root: str, dst_root: str) -> None:
```

Comment:
1. there's a recursive copy command in fsspec
2. please use our filesystem source! it has very well tested glob and standardized handling of timestamps. this also lets you write local tests